### ISSUE TITLE
Interfacing Ultrasonic sensors with Beaglebone Black Wireless

### ISSUE THREAD

**Issue by joyalraju** [joyalraju/Capstone-Project-Summer-2020#10]

**Comment by mikeshams** [comment]
1. What issues did you or others might face with connecting the Ultrasonic Sensors to the BBB Wireless in terms of Hardware or Software?
2. Did you use dbt files? Why not?

**Comment by merlinthankachan** [comment]
1. While interfacing ultrasonic sensor it was inevitable to provide delay to trigger  the pin and to receive pulse back from the echo pin. So, the need to give enough delay was a concern that we faced while interfacing ultrasonic sensor with bbb wireless. Also, the bbb may get damaged if one is not making sure that the output from echo pin is less than 3.3 V. These are the issues we have faced while interfacing ultrasonic sensors with bbb wireless.

2. We didn't use dtb (device tree binary) files right now. This is dts (device tree source) code which is just rough allocation and thus the codes can be simplified using iobb header files. If we are simplifying similarly and changed to dtb file it will be a simple code. So, we are planning to compiling  this as a whole code to get dtb file along with the code for lcd, motor and so on at the last part of our milestones.
